test-harness/createMasterFiles.ts

```diff
@@ -27,6 +27,4 @@ async function recordMasterFile({ path, method, headers, body }) {
   }
 }
 
-(async function () {
-  await Promise.all(requests.map(recordMasterFile));
-})();
+Promise.all(requests.map(recordMasterFile));
```

Comment:
curious, does it not end the execution of `test-harness/createMasterFiles.ts` prematurely now (does it save all the files to be saved)?

Reply:
so the way it is now (without `async/await`), will `node` wait for the promises to finish?

Reply:
Correct! Node will always wait for all the open handles before closing the process. 

```javascript
new Promise(resolve => {
  setTimeout(resolve,5000);
})
```

If you put this in a `test.js` and you execute it, you'll see it will wait before exiting the process.

Reply:
ha! just tested it :) you learn something new every day